<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MediaControl</class>
 <widget class="QWidget" name="MediaControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>69</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSlider" name="horizontalSliderTime">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelTime"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButtonStop">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="softprojector.qrc">
         <normaloff>:/icons/icons/stop.png</normaloff>:/icons/icons/stop.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPlayPause">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="softprojector.qrc">
         <normaloff>:/icons/icons/play.png</normaloff>:/icons/icons/play.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonMute">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="softprojector.qrc">
         <normaloff>:/icons/icons/speaker.png</normaloff>:/icons/icons/speaker.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSliderVolume">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="softprojector.qrc"/>
 </resources>
 <connections/>
</ui>
